docs/operations/metrics.md

```diff
@@ -323,6 +323,8 @@ decisions.
 |`segment/usedPercent`|Percentage of space used by served segments.|dataSource, tier, priority.|< 100%|
 |`segment/count`|Number of served segments.|dataSource, tier, priority.|Varies.|
 |`segment/pendingDelete`|On-disk size in bytes of segments that are waiting to be cleared out|Varies.|
+| `segment/averageRowCount`       | The average number of rows per segment in a datasource               | dataSource, tier, priority.                 | ~5 million |
```

Comment:
```suggestion
| `segment/rows/avg`| The average number of rows per segment on a historical| dataSource, tier, priority.| ~5 million |
```

I think this name is closer to the pattern in other metrics that report min + max eg.`jetty/threadpool/` `jvm/*/max` etc.